x-pack/metricbeat/module/cockroachdb/status/status_integration_test.go

```diff
@@ -31,8 +31,8 @@ func init() {
 func TestFetch(t *testing.T) {
 	service := compose.EnsureUp(t, "cockroachdb")
 
-	f := mbtest.NewReportingMetricSetV2(t, getConfig(service.Host()))
-	events, errs := mbtest.ReportingFetchV2(f)
+	f := mbtest.NewReportingMetricSetV2Error(t, getConfig(service.Host()))
+	events, errs := mbtest.ReportingFetchV2Error(f)
```

Comment:
It could be good to use here these more generic methods, in case this code is used as example for other light modules.
```
f := mbtest.NewFetcher(t, getConfig(service.Host()))
events, errs := f.FetchEvents()
```